tests/common/fixtures/advanced_reboot.py

```diff
@@ -168,7 +168,14 @@ def __buildTestbedData(self, tbinfo):
         self.vlanMaxCnt = len(self.mgFacts['minigraph_vlans'].values()[0]['members']) - 1
 
         self.rebootData['dut_hostname'] = self.mgFacts['minigraph_mgmt_interface']['addr']
+        config_facts = self.duthost.get_running_config_facts()
+        vlan_table = config_facts['VLAN']
+        vlan_name = list(vlan_table.keys())[0]
```

Comment:
Will vlan name always be there?

Reply:
Changed single tor to keep use router mac, as for dual tor seems like at least one vlan will always be there